contents/handbook/small-teams/replay/index.mdx

```diff
@@ -16,22 +16,35 @@ We build the [heads-up display](https://en.wikipedia.org/wiki/Head-up_display) w
 
 ## Q4 2023 Goals
 
-### 🎯 Goal 1 : Project "NoteForce Exploration 3000" (Ben, David)
+### 🎯 Goal 1: 10 Happy Android Replay Users 
 
-Why? Pairing up with Michael, Thomas, and Cory from Team Product Analytics to focus on not just releasing Notebooks, but integrating them seamlessly with PostHog 3000 to make it a great experience to explore any and all data across PostHog.
+We'll also release an alpha of iOS Mobile Replay.
 
-### 🎯 Goal 2 : Mobile Session Replay (Manoel, Paul)
+Why? 
 
-Why? - The people want it and we have the right skills in house to do a good job of this. Replay is the main focus, but making mobile a first-class citizen of PostHog is the real goal.
+We've done the work's base and now we want to release the new Android SDK with session replay and onboard customers to validate the product. 
 
-Follow along in [our Q4 MegaIssue](https://github.com/PostHog/posthog/issues/17803)
```

Comment:
Do you not want to link to the latest mega-issue anymore?

Reply:
Ah, just forgot :)